### ISSUE TITLE
Submit Implementation -00

### ISSUE THREAD

**Issue by tfpauly** [taps-api/drafts#129]
Issue to track submitting -00 draft of implementation. We need to:

- Sweep the remaining issues for sections that need more text
- Nits
- Editorial review for flow, etc.

**Comment by theri** [comment]
Would it be possible to still merge the intents-implementation branch #122 for -00, please? 
We moved the Intents to the Appendix as requested and we think we addressed all the blocking comments. However, I have to admit that the web interface for that Pull Request is a bit confusing for me, so I may have missed something...

**Comment by tfpauly** [comment]
Yes, sure we can definitely merge that first! I'll get to reviewing that today, and will merge/edit as needed.

**Comment by tfpauly** [comment]
Merged, and finished writing remaining issues. We now need to the passes for nits and editorial review.

**Comment by abrunstrom** [comment]
thanks @tfpauly for filling the remaining issues! I will go through the whole document tomorrow and polish nits and anything else I find. (Just back from project meeting last night and fully booked today unfortunately.)

**Comment by abrunstrom** [comment]
Did not make it through the whole document tonight, will continue tomorrow.

**Comment by tfpauly** [comment]
Sounds good. Anna, can you do the submission when ready?

**Comment by abrunstrom** [comment]
Sure, I can do that. Do you also want to review it before I submit?

**Comment by tfpauly** [comment]
I'll edit if I see things that are particularly off, but I'm not going to have too much time before the deadline. Please feel free to take the editing from here!

**Comment by abrunstrom** [comment]
Ok! I have now completed my editing pass. I think it is fine for -00. I will submit it tomorrow, in case anyone has last comments.

**Comment by tfpauly** [comment]
Looks good to me! Ship it.
